Squaads/Domain-Driven-Hexagon
# feat: cleanup template

## **Problem**

La template tiene un ejemplo en la rama main, esto hace que al empezar un nuevo proyecto haya que perder tiempo en limpiar esta rama. Sugiero limpiar esta rama y dejar solo lo estrictamente necesario.

- [] No hay rastro del ejemplo

## **Changelog**

- Se ha borrado el ejemplo, y se ha creado una rama con el ejemplo, llamada "example".

## **How to test**

The steps to reproduce this PR and to check its functionality would be the following:

1.  Review the code expecting to not see the example

## **Type of change (Optional)**

- [ ] Refactor (non-breaking change which modifies code keeping the same functionality)
